packages/lesswrong/components/posts/TableOfContents/TableOfContentsRow.tsx

```diff
@@ -102,15 +102,15 @@ const styles = (theme: ThemeType) => ({
     minHeight: 70,
     display: 'flex',
     flexDirection: 'column-reverse',
-    transition: 'opacity 0.4s ease-in-out, max-height 0.4s ease-in-out',
+    transition: 'opacity 0.4s ease-in-out, max-height 0.4s ease-in-out, margin-top 0.4s ease-in-out',
   },
   '@global': {
     // Hard-coding this class name as a workaround for one of the JSS plugins being incapable of parsing a self-reference ($titleContainer) while inside @global
     [`body:has(.headroom--pinned) .${TITLE_CONTAINER_CLASS_NAME}`]: {
       opacity: 0,
     },
     [`body:has(.headroom--unfixed) .${TITLE_CONTAINER_CLASS_NAME}`]: {
-      opacity: 0,
+      marginTop: 64
```

Comment:
Checked that this works on both review winner posts and regular posts?

Reply:
Yep, I checked

Reply:
It makes an animation very very slightly more janky, but probably not in a way that's noticeable to anyone but me